### YamlMime:ManagedReference
items:
- uid: com.microsoft.azure.batch._create_tasks_error_exception
  id: _create_tasks_error_exception
  parent: com.microsoft.azure.batch
  children:
  - com.microsoft.azure.batch._create_tasks_error_exception.CreateTasksErrorException(final String,List<TaskAddResult>,List<TaskAddParameter>)
  - com.microsoft.azure.batch._create_tasks_error_exception.failureTaskList()
  - com.microsoft.azure.batch._create_tasks_error_exception.pendingTaskList()
  href: com.microsoft.azure.batch._create_tasks_error_exception.yml
  langs:
  - java
  name: CreateTasksErrorException
  nameWithType: CreateTasksErrorException
  fullName: com.microsoft.azure.batch.CreateTasksErrorException
  type: Class
  source:
    remote: &o0
      path: src/main/java/com/microsoft/azure/batch/CreateTasksErrorException.java
      branch: master
      repo: https://github.com/Azure/azure-batch-sdk-for-java
    path: src/main/java/com/microsoft/azure/batch/CreateTasksErrorException.java
    startLine: 19
  package: com.microsoft.azure.batch
  summary: <p>The exception that is thrown when the <xref uid="" data-throw-if-not-resolved="false">TaskOperations#createTasks(String, List)</xref> operation is terminated. </p>
  syntax: &o1
    content: public class CreateTasksErrorException extends BatchErrorException
  inheritance:
  - java.lang.Object
  - RestException
  - com.microsoft.azure.batch.protocol.models._batch_error_exception
  inheritedMembers:
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String,final Response<ResponseBody>)
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String,final Response<ResponseBody>,final BatchError)
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.body()
- uid: com.microsoft.azure.batch._create_tasks_error_exception.CreateTasksErrorException(final String,List<TaskAddResult>,List<TaskAddParameter>)
  id: CreateTasksErrorException(final String,List<TaskAddResult>,List<TaskAddParameter>)
  parent: com.microsoft.azure.batch._create_tasks_error_exception
  href: com.microsoft.azure.batch._create_tasks_error_exception.yml
  langs:
  - java
  name: CreateTasksErrorException(final String message, List<TaskAddResult> failureTaskList, List<TaskAddParameter> pendingTaskList)
  nameWithType: CreateTasksErrorException.CreateTasksErrorException(final String message, List<TaskAddResult> failureTaskList, List<TaskAddParameter> pendingTaskList)
  fullName: com.microsoft.azure.batch.CreateTasksErrorException.CreateTasksErrorException(final String message, List<TaskAddResult> failureTaskList, List<TaskAddParameter> pendingTaskList)
  overload: com.microsoft.azure.batch._create_tasks_error_exception.CreateTasksErrorException*
  type: Constructor
  source:
    remote: *o0
    path: src/main/java/com/microsoft/azure/batch/CreateTasksErrorException.java
    startLine: 28
  package: com.microsoft.azure.batch
  summary: >-
    <p>Initializes a new instance of the <xref uid="com.microsoft.azure.batch._create_tasks_error_exception" data-throw-if-not-resolved="false">CreateTasksErrorException</xref> class.</p>

    <p></p>
  syntax:
    content: public CreateTasksErrorException(final String message, List<TaskAddResult> failureTaskList, List<TaskAddParameter> pendingTaskList)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>The exception message. </p>
    - id: failureTaskList
      type: 5618da2dcom.microsoft.azure.batch.protocol.models._task_add_resulta08ddfce
      description: <p>The list of <xref uid="" data-throw-if-not-resolved="false">TaskAddResult</xref> instances containing failure details for tasks that were not successfully created. </p>
    - id: pendingTaskList
      type: 5618da2dcom.microsoft.azure.batch.protocol.models._task_add_parametera08ddfce
      description: <p>The list of <xref uid="" data-throw-if-not-resolved="false">TaskAddParameter</xref> instances containing the tasks that were not added, but for which the operation can be retried. </p>
- uid: com.microsoft.azure.batch._create_tasks_error_exception.failureTaskList()
  id: failureTaskList()
  parent: com.microsoft.azure.batch._create_tasks_error_exception
  href: com.microsoft.azure.batch._create_tasks_error_exception.yml
  langs:
  - java
  name: failureTaskList()
  nameWithType: CreateTasksErrorException.failureTaskList()
  fullName: List<TaskAddResult> com.microsoft.azure.batch.CreateTasksErrorException.failureTaskList()
  overload: com.microsoft.azure.batch._create_tasks_error_exception.failureTaskList*
  type: Method
  source:
    remote: *o0
    path: src/main/java/com/microsoft/azure/batch/CreateTasksErrorException.java
    startLine: 41
  package: com.microsoft.azure.batch
  syntax:
    content: public List<TaskAddResult> failureTaskList()
    return:
      type: 5618da2dcom.microsoft.azure.batch.protocol.models._task_add_resulta08ddfce
      description: <p>The list of <xref uid="" data-throw-if-not-resolved="false">TaskAddResult</xref> instances containing failure details for tasks that were not successfully created. </p>
- uid: com.microsoft.azure.batch._create_tasks_error_exception.pendingTaskList()
  id: pendingTaskList()
  parent: com.microsoft.azure.batch._create_tasks_error_exception
  href: com.microsoft.azure.batch._create_tasks_error_exception.yml
  langs:
  - java
  name: pendingTaskList()
  nameWithType: CreateTasksErrorException.pendingTaskList()
  fullName: List<TaskAddParameter> com.microsoft.azure.batch.CreateTasksErrorException.pendingTaskList()
  overload: com.microsoft.azure.batch._create_tasks_error_exception.pendingTaskList*
  type: Method
  source:
    remote: *o0
    path: src/main/java/com/microsoft/azure/batch/CreateTasksErrorException.java
    startLine: 48
  package: com.microsoft.azure.batch
  syntax:
    content: public List<TaskAddParameter> pendingTaskList()
    return:
      type: 5618da2dcom.microsoft.azure.batch.protocol.models._task_add_parametera08ddfce
      description: <p>The list of <xref uid="" data-throw-if-not-resolved="false">TaskAddParameter</xref> instances containing the tasks that were not added, but for which the operation can be retried. </p>
references:
- uid: 3d93591e
  spec.java:
  - name: final String
    fullName: final String
- uid: 5618da2dcom.microsoft.azure.batch.protocol.models._task_add_resulta08ddfce
  spec.java:
  - name: List<
    fullName: List<
  - uid: com.microsoft.azure.batch.protocol.models._task_add_result
    name: TaskAddResult
    fullName: com.microsoft.azure.batch.protocol.models.TaskAddResult
    href: com.microsoft.azure.batch.protocol.models._task_add_result.yml
  - name: '>'
    fullName: '>'
- uid: 5618da2dcom.microsoft.azure.batch.protocol.models._task_add_parametera08ddfce
  spec.java:
  - name: List<
    fullName: List<
  - uid: com.microsoft.azure.batch.protocol.models._task_add_parameter
    name: TaskAddParameter
    fullName: com.microsoft.azure.batch.protocol.models.TaskAddParameter
    href: com.microsoft.azure.batch.protocol.models._task_add_parameter.yml
  - name: '>'
    fullName: '>'
- uid: com.microsoft.azure.batch._create_tasks_error_exception.CreateTasksErrorException*
  name: CreateTasksErrorException
  nameWithType: CreateTasksErrorException.CreateTasksErrorException
  fullName: com.microsoft.azure.batch.CreateTasksErrorException.CreateTasksErrorException
  package: com.microsoft.azure.batch
- uid: com.microsoft.azure.batch._create_tasks_error_exception.failureTaskList*
  name: failureTaskList
  nameWithType: CreateTasksErrorException.failureTaskList
  fullName: List<TaskAddResult> com.microsoft.azure.batch.CreateTasksErrorException.failureTaskList
  package: com.microsoft.azure.batch
- uid: com.microsoft.azure.batch._create_tasks_error_exception.pendingTaskList*
  name: pendingTaskList
  nameWithType: CreateTasksErrorException.pendingTaskList
  fullName: List<TaskAddParameter> com.microsoft.azure.batch.CreateTasksErrorException.pendingTaskList
  package: com.microsoft.azure.batch
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String,final Response<ResponseBody>)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException(final String message, final Response<ResponseBody> response)
  nameWithType: BatchErrorException.BatchErrorException(final String message, final Response<ResponseBody> response)
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException.BatchErrorException(final String message, final Response<ResponseBody> response)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the <xref uid="com.microsoft.azure.batch.protocol.models._batch_error_exception" data-throw-if-not-resolved="false">BatchErrorException</xref> class.</p>

    <p></p>
  syntax:
    content: public BatchErrorException(final String message, final Response<ResponseBody> response)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message or the response content if a message is not available </p>
    - id: response
      type: 4f402e61
      description: <p>the HTTP response </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String,final Response<ResponseBody>,final BatchError)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException(final String message, final Response<ResponseBody> response, final BatchError body)
  nameWithType: BatchErrorException.BatchErrorException(final String message, final Response<ResponseBody> response, final BatchError body)
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException.BatchErrorException(final String message, final Response<ResponseBody> response, final BatchError body)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the <xref uid="com.microsoft.azure.batch.protocol.models._batch_error_exception" data-throw-if-not-resolved="false">BatchErrorException</xref> class.</p>

    <p></p>
  syntax:
    content: public BatchErrorException(final String message, final Response<ResponseBody> response, final BatchError body)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message or the response content if a message is not available </p>
    - id: response
      type: 4f402e61
      description: <p>the HTTP response </p>
    - id: body
      type: 93de0ef9com.microsoft.azure.batch.protocol.models._batch_error
      description: <p>the deserialized response body </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.body()
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: body()
  nameWithType: BatchErrorException.body()
  fullName: BatchError com.microsoft.azure.batch.protocol.models.BatchErrorException.body()
  type: Method
  syntax:
    content: public BatchError body()
    return:
      type: com.microsoft.azure.batch.protocol.models._batch_error
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception
  parent: com.microsoft.azure.batch.protocol.models
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException
  nameWithType: BatchErrorException
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException
  type: Class
  summary: <p>Exception thrown for an invalid response with <xref uid="com.microsoft.azure.batch.protocol.models._batch_error" data-throw-if-not-resolved="false">BatchError</xref> information. </p>
  syntax:
    content: public class BatchErrorException
- uid: com.microsoft.azure.batch.protocol.models._task_add_result
  parent: com.microsoft.azure.batch.protocol.models
  href: com.microsoft.azure.batch.protocol.models._task_add_result.yml
  name: TaskAddResult
  nameWithType: TaskAddResult
  fullName: com.microsoft.azure.batch.protocol.models.TaskAddResult
  type: Class
  summary: <p>Result for a single task added as part of an add task collection operation. </p>
  syntax:
    content: public class TaskAddResult
- uid: com.microsoft.azure.batch.protocol.models._task_add_parameter
  parent: com.microsoft.azure.batch.protocol.models
  href: com.microsoft.azure.batch.protocol.models._task_add_parameter.yml
  name: TaskAddParameter
  nameWithType: TaskAddParameter
  fullName: com.microsoft.azure.batch.protocol.models.TaskAddParameter
  type: Class
  summary: <p>An Azure Batch task to add. </p>
  syntax:
    content: public class TaskAddParameter
- uid: com.microsoft.azure.batch._create_tasks_error_exception
  parent: com.microsoft.azure.batch
  href: com.microsoft.azure.batch._create_tasks_error_exception.yml
  name: CreateTasksErrorException
  nameWithType: CreateTasksErrorException
  fullName: com.microsoft.azure.batch.CreateTasksErrorException
  type: Class
  summary: <p>The exception that is thrown when the <xref uid="" data-throw-if-not-resolved="false">TaskOperations#createTasks(String, List)</xref> operation is terminated. </p>
  syntax: *o1
